data_collection/other_integrations/web_log_analytics.rst

```diff
@@ -101,3 +105,29 @@ Supported log formats:
 - log files of some popular Cloud Saas services: Amazon CloudFront logs, Amazon S3 logs
 - streaming media server log files such as: Icecast
 - log files with and without the virtual host will be imported
+
+The script does not directly support importing logs from log aggregator tools, like Grafana Loki or ELK. If you'd like to import logs from one of those, you need to download them to the disk first.
+
+
+Performance considerations & rate limiting
+----------------------
+
+The script needs CPU to read and parse the log files, but it is usually Piwik PRO server itself which will limit the import speed due to network latency.
+To improve performance, you can use the **--recorders** option to specify the number of parallel threads which will import hits into Piwik PRO. We recommend to increase **--recorders=N** until import speed is satisfying for you.
+
+If you are Piwik PRO Core user, please make sure, that you are not hitting rate limits, by using **--debug-force-one-hit-every-Ns** flag to slow down the import process.
+
+
+
+Using CRON to automatically push the logs every day
+----------------------
+
+Your logs should be automatically rotated and stored on your webserver, for instance in daily logs
+*/var/log/apache/access-%Y-%m-%d.log* (where %Y, %m and %d represent the year,
+month and day).
+
+If so, you can import your logs automatically each day (at 0:30), by using a cron command:
+
+.. code-block:: cron
+
+        30 0 * * * /wherever-you-cloned-the-repo/log-analytics/import-logs.py -u example.piwik.pro `date --date=yesterday +/var/log/apache/access-\%Y-\%m-\%d.log`
```

Comment:
`-u`? I'm not familiar with such option for the `log_import` script.
Besides, I won't oppose if you remove this whole section.

Reply:
removed